setup.py

```diff
@@ -11,7 +11,7 @@
 optional_dependencies = {
     "docs": ["sphinx_mdolab_theme"],
     "test": ["pytest", "pytest-cov", "coverage"],
-    "ffd": ["pygeo>=1.6.0"]
+    "ffd": ["pygeo>=1.6.0"],

```

Comment:
pygeo>=1.12.2 ?

Reply:
Good catch. pyGeo 1.12.2 was the latest that works with the old versions of the other packages, but older versions of pyGeo work too. I'll see if 1.6.0 works with the old build and if so leave that as the old build version.